pceplib/pcep.h

```diff
@@ -45,4 +43,12 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <pthread.h>
+
+/* Cross-compilation seems to have trouble finding this */
+#if defined(TCP_MD5SIG_MAXKEYLEN)
+#  define PCEP_MD5SIG_MAXKEYLEN TCP_MD5SIG_MAXKEYLEN
```

Comment:
Can we stay consistent with the rest of the defines in here and not put a space between the # and the define keyword

Reply:
done